### ISSUE TITLE
Creating a multi-page form, continuing to next page does not go to top

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/platform-forms-client#4123]
## Description
When previewing or testing a multi-page form with lots of content, clicking continue to go to the next page / step does not bring you to the top of the page (page title) as expected.

## Steps to reproduce
1. Go to GC Forms
2. Create 3 new pages with 3 questions each
3. Go to test/preview
4. Click "Continue" through the pages
5. Notice focus down near the bottom element

## Details
- **Browser**: Edge
- **Operating system**: Windows
- **Environment**: Staging
- **Language**: French
- **Viewport**: Tablet

## Expected behaviour
User should be taken to top of next page with a focus on the page title, when filling out form, and when previewing the form fillout.

## Screenshots or videos

![test](https://github.com/user-attachments/assets/c77a4281-087c-4f9f-9218-5807bf768d0b)
